docs/astro/sql/operators/raw_sql.rst

```diff
@@ -0,0 +1,27 @@
+====================
+run_raw_sql operator
+====================
+
+It allows users to declare any SQL statement using the Astro SDK templating available in transform. By default this operator returns ``None`` but it also empowers users to define the task output (e.g. a list with the row results, etc) by using the handler argument.
```

Comment:
Can we add a heading `When to use the append operator` just be in sync with other operators doc - https://github.com/astronomer/astro-sdk/blob/main/docs/astro/sql/operators/append.rst

Reply:
Added